src/сomponents/TodoItem.tsx

```diff
@@ -0,0 +1,56 @@
+import { useEffect, useState } from 'react';
+import classNames from 'classnames';
+import { Todo } from '../types/Todo';
+
+type Props = {
+  todo: Todo;
+  todosIdToDelete: number[];
+  setTodosIdToDelete: (todosId: number[]) => void;
+};
+
+export const TodoItem: React.FC<Props> = ({
+  todo,
+  todosIdToDelete,
+  setTodosIdToDelete,
+}) => {
+  const [markLoading, setMarkLoading] = useState(false);
+
+  useEffect(() => {
+    if (todosIdToDelete.includes(todo.id)) {
+      setMarkLoading(true);
+    } else {
+      setMarkLoading(false);
+    }
+  }, [todosIdToDelete]);
+
+  return (
+    <div className="todo">
+      <label className="todo__status-label">
+        <input
+          type="checkbox"
+          className="todo__status"
+        />
+      </label>
+
+      <span className="todo__title">{todo.title}</span>
+      <button
+        type="button"
+        className="todo__remove"
+        onClick={() => setTodosIdToDelete([...todosIdToDelete, todo.id])}
```

Comment:
pls use [prevState](https://blog.greenroots.info/react-hook-usestate-lazy-initialization-previous-state#heading-how-to-get-the-previous-state-data-and-use-it) for such cases

Reply:
i wanted to do this, but I couldn't type it )
who would have thought it could be: Dispatch<SetStateAction<number[]>>